meta-networking/recipes-support/libconfuse/libconfuse_3.3.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "libConfuse is a configuration file parser library"
+LICENSE = "CUSTOM"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=42fa47330d4051cd219f7d99d023de3a"
+
+DEPENDS = "gettext-native"
+
+#SRCREV = "1b3f11bde315e221474f7d066ce4efb4ff4d39e3"
+SRC_URI = "git://github.com/libconfuse/libconfuse.git;branch=master;tag=v3.3"
+
+S = "${WORKDIR}/git"
+
+inherit autotools pkgconfig
+
+do_configure_prepend(){
+    cd ${S}
+    ${S}/autogen.sh
+}
+
+do_compile(){
+    cd ${S}
+    oe_runmake
+}
+
+do_install(){
+    cd ${S}
+    oe_runmake DESTDIR=${D} install
+}
```

Comment:
this also should go away

Reply:
Yes, I could delete it once I added B = "${S}".